http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20231002_drop_columns_in_nodepool_table.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="qi" id="qi_drop_columns_in_nodepool_table">
+        <dropColumn tableName="NODEPOOL" columnName="dateAccessed"/>
+        <addColumn tableName="APP">
+            <column name="namespace" type="VARCHAR(254)">
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+
+        <dropForeignKeyConstraint baseTableName="APP" constraintName="FK_APP_NAMESPACE_ID" />
+        <dropNotNullConstraint tableName="APP" columnName="namespaceId" columnDataType="BIGINT" />
+    </changeSet>
+
+    <changeSet logicalFilePath="leonardo" id="qi_update_app_namespace" author="qi">
+        <sql>
+            UPDATE APP LEFT JOIN NAMESPACE ns ON ns.id = APP.namespaceId
```

Comment:
Could this be a `JOIN` instead of `LEFT JOIN`? I think all apps will have a namespaceId foreign key.

Reply:
Yea, I think so..I used `LEFT JOIN` just to more forgiving...

Reply:
Yeah LEFT JOIN is more forgiving.. no downside I guess (it could just leave `null` records if there are any problems).

It's OK with me if there will be a followup PR to add a not-null constraint, etc.